src/Repository/CardRepository.php

```diff
@@ -23,4 +23,9 @@ public function findOneByName(string $name): ?Card
     {
         return $this->entityManager->getRepository(Card::class)->findOneBy(['name' => $name]);
     }
+
+    public function findOneById(int $id): ?Card
```

Comment:
IMHO better approach for fetching entity from db will be by using `getById(int): Card`. We assume that if user want to fetch something by id from our system then that should exist. In not then throw execption for it. That allow us to avoid null checking after using this method.

Reply:
done